addons/mail/static/src/models/follower_list_menu_view.js

```diff
@@ -43,7 +43,7 @@ registerModel({
          * @returns {Boolean}
          */
         _computeIsDisabled() {
-            return this.chatterOwner.isDisabled;
+            return !this.chatterOwner.hasWriteAccess;
```

Comment:
Viewing followers should probably be linked to read access not write?

Reply:
Or I guess disabled is doing something else in this context?

Reply:
Nope, you right!